Framework/PythonInterface/mantid/api/src/Exports/ITableWorkspace.cpp

```diff
@@ -66,8 +66,8 @@ namespace {
 
 /// Boost macro for "looping" over builtin types
 #define BUILTIN_TYPES                                                          \
-  BOOST_PP_TUPLE_TO_LIST(                                                      \
-      7, (double, std::string, int, uint32_t, int64_t, float, uint64_t))
+  BOOST_PP_TUPLE_TO_LIST(7, (double, std::string, int, size_t, uint32_t,       \
```

Comment:
I think you need to bump the `7` to an `8` or this will knock off the converter for the last one in the list.

Reply:
Oh! Ok will do.